src/NodaTime/CalendarSystem.cs

```diff
@@ -602,6 +603,9 @@ internal Era GetEra([Trusted] int absoluteYear)
         /// </summary>
         /// <param name="yearMonthDay">The value to validate.</param>
         [Conditional("DEBUG")]
+#if NET45
+        [ExcludeFromCodeCoverage]
+#endif
         internal void DebugValidateYearMonthDay(YearMonthDay yearMonthDay)
         {
             // Avoid the line even being compiled in a release build...
```

Comment:
(Incidentally, what does this mean?)

Reply:
The attribute isn't available in netstandard, but that's okay, because we only run coverage tests under net45. It's used in the coverage filter to detect methods we don't record coverage for.